app/mender.go

```diff
@@ -219,6 +223,18 @@ func (m *Mender) FetchUpdate(url string) (io.ReadCloser, int64, error) {
 	return m.updater.FetchUpdate(m.download, url, m.GetRetryPollInterval())
 }
 
+func filterArtifactDepends(depends map[string]interface{}) map[string]interface{} {
```

Comment:
Filtered for what?

Reply:
Filter `device_type`. This was a poor name, thanks for noticing.

In any case I am [removing the method](https://github.com/mendersoftware/mender/pull/1024/files#r948825773).